packages/core/src/blockchain/Blockchain.ts

```diff
@@ -76,7 +77,10 @@ export default class Blockchain implements IBlockchainApi {
 
   public constructor(api: ApiPromise) {
     this.api = api
-    this.portablegabi = new gabi.Blockchain('portablegabi', this.api as any)
+    this.portablegabi = new gabi.Blockchain(
+      'portablegabi',
+      this.api as ApiPromise & IPortablegabiApi<PgabiModName>
+    )
```

Comment:
this is not going to be around for long, hehe

Reply:
Ohh yes true, this is only a typing fix.